### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by dheerajck**
Updated readme

Added examples of WRatio, QRatio and updated score values
Added examples of string preprocessing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,39 +89,84 @@ pip install .
 ```
 
 ## Usage
-Some simple functions are shown below. A complete documentation of all functions can be found [here](https://maxbachmann.github.io/RapidFuzz/Usage/index.html).
+Some simple functions are shown below. A complete documentation of all functions can be found [here](https://maxbachmann.github.io/RapidFuzz/Usage/index.html).<br>
+Note that from RapidFuzz 3.0.0, strings are not preprocessed(removing all non alphanumeric characters, trimming whitespaces, converting all characters to lower case) by default. Which means that when comparing two strings that have the same characters but different cases("this is a word", "THIS IS A WORD") their similarity score value might be different, so when comparing such strings you might see a difference in score value compared to previous versions. Some examples of string matching with preprocessing can be found [here](https://github.com/dheerajck/RapidFuzz#weighted-ratio).
```
Comment (maxbachmann) [inline README.md]:
```suggestion
Note that from RapidFuzz 3.0.0, strings are not preprocessed(removing all non alphanumeric characters, trimming whitespaces, converting all characters to lower case) by default. Which means that when comparing two strings that have the same characters but different cases("this is a word", "THIS IS A WORD") their similarity score value might be different, so when comparing such strings you might see a difference in score value compared to previous versions. Some examples of string matching with preprocessing can be found [here](#weighted-ratio).
```

This should make the reference relative in the current document.

**Comment by dheerajck** [comment]
Do you think the name processor can be confusing and something like string_preprocessor would be a better name

**Comment by dheerajck** [comment]
Do you think that the parameter name processor can be confusing, and something like string_preprocessor would be a better name ??

**Comment by dheerajck** [comment]
Dont you think that the parameter name processor can be confusing, and something like string_preprocessor would be a better name ??

**Comment by maxbachmann** [comment]
> Dont you think that the parameter name processor can be confusing, and something like string_preprocessor would be a better name ??

I agree it is not a perfect name. The naming stems from `fuzzywuzzy` using the named argument `processor` in their `process.*` APIs. I added the argument to every scorer, which in hindsight wasn't a great idea. It saves the user very little typing:
```python
Levenshtein.distance(s1, s2, processor=utils.default_process)
```
vs
```python
Levenshtein.distance(utils.default_process(s1), utils.default_process(s2))
```
in addition the performance difference is pretty small. For short sequences <16 characters the second implementation appears a couple percent faster and for longer ones calling it internally appears to be around 10% faster. So it only makes a difference when working with very fast scorers like Prefix/Postfix/Hamming and long sequences. Even then when comparing multiple sequences your better off using the scorer with the `process.*` APIs.

For the `process.*` APIs that is a different story, since:
1) it saves more typing
2) I am able to call the preprocessing function in a more performant way

For these reasons I was actually playing with the thought of deprecating the `processor` argument in scorers.

**Pull request merged**